test/e2e/network/conntrack.go

```diff
@@ -268,4 +269,26 @@ var _ = SIGDescribe("Conntrack", func() {
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	ginkgo.AfterEach(func() {
+		// if ginkgo.CurrentGinkgoTestDescription().Failed {
+		// Dump conntrack table of each node for troubleshooting using the kube-proxy pods
+		namespace := "kube-system"
+		pods, err := fr.ClientSet.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{})
```

Comment:
why is this panicking?
```
test/e2e/network/conntrack.go:273
Test Panicked
GOROOT/src/runtime/panic.go:212

Panic: runtime error: invalid memory address or nil pointer dereference

Full stack:
k8s.io/kubernetes/test/e2e/network.glob..func1.5()
	test/e2e/network/conntrack.go:277 +0x3e
```

Reply:
The only thing that comes to my mind is that fr is nil...

Reply:
ginkgo ?? :man_shrugging: